examples/storybook/src/stories/Breadcrumb.stories.ts

```diff
@@ -0,0 +1,18 @@
+import { storiesOf } from '@storybook/angular';
+import { wrapWithHits } from '../wrap-with-hits';
+import meta from '../meta';
+
+storiesOf('Breadcrumb', module)
+  .addDecorator(meta)
+  .add('default', () => ({
+    component: wrapWithHits({
+      template: ` <ais-breadcrumb [attributes]="[ 'hierarchicalCategories.lvl0', 'hierarchicalCategories.lvl1', 'hierarchicalCategories.lvl2' ]" > </ais-breadcrumb> `,
+      searchParameters: {
+        hierarchicalFacetsRefinements: {
```

Comment:
It would be nice to add a `HierarchicalMenu` to the playground otherwise it's a bit hard to correctly see how the `Breadcrumb` widget is working rather than use static `searchParameters` (see comment about `wrap-with-hits`).

Reply:
done!
I kept the static parameter as it initialises the breadcrumb with a specific value and conveys better the look